manifest.yml

```diff
@@ -1,6 +1,6 @@
 ---
 applications:
-- name: govuk-paas
+- name: paas-product-page
```

Comment:
We need to duplicate this name in the `hosts` array at the bottom so that the newly deployed app also gets a route of `paas-product-page.${APP_DOMAIN}`.

It seems that the existing app in production has both routes, because it was originally deployed as `govuk-paas` with no additional routes and then we added a route in 9b57c6d6a5a0fce83c9b59ba93ea617f0a8f4117:
```
➜  paas-cf git:(master) ✗ cf app govuk-paas
Showing health and status for app govuk-paas in org govuk-paas / space docs as dan.carley@digital.cabinet-office.gov.uk...
OK

requested state: started
instances: 2/2
usage: 64M x 2 instances
urls: govuk-paas.cloudapps.digital, paas-roadmap.cloudapps.digital
```

CF will add new routes listed in `hosts`, but it won't remove them and it won't include the app's name. So if you push the app as a new app you get:
```
➜  paas-product-page git:(master) cf push
Using manifest file /Users/dcarley/projects/paas/paas-product-page/manifest.yml

Creating app govuk-paas in org govuk-paas / space docs as admin...
OK
…
Showing health and status for app govuk-paas in org govuk-paas / space docs as admin...
OK

requested state: started
instances: 2/2
usage: 64M x 2 instances
urls: paas-roadmap.dcarley.dev.cloudpipelineapps.digital
```